corporate/views.py

```diff
@@ -161,17 +171,70 @@ def initial_upgrade(request: HttpRequest) -> HttpResponse:
     response = render(request, 'corporate/upgrade.html', context=context)
     return response
 
+@has_request_variables
+def sponsorship(request: HttpRequest, user: UserProfile,
+                organization_type: str=REQ("organization-type", validator=check_string),
+                website: str=REQ("website", validator=check_string),
+                description: str=REQ("description", validator=check_string)) -> HttpResponse:
+    realm = user.realm
+
+    requested_by = user.full_name
+
+    user_role = ""
+    if user.role == UserProfile.ROLE_REALM_OWNER:
+        user_role = "Realm owner"
+    elif user.role == UserProfile.ROLE_REALM_ADMINISTRATOR:
+        user_role = "Realm adminstrator"
+    elif user.role == UserProfile.ROLE_MEMBER:
+        user_role = "Member"
+    elif user.role == UserProfile.ROLE_GUEST:
+        user_role = "Guest"
+
+    if user.is_billing_admin:
+        user_role = user_role + "/Billing admin"
+
+    support_url = ""
+    if hasattr(settings, "STAFF_SUBDOMAIN"):
+        support_realm_uri = get_realm(settings.STAFF_SUBDOMAIN).uri
+        support_url = urljoin(support_realm_uri, urlunsplit(("", "", reverse('analytics.views.support'),
+                              urlencode({"q": realm.string_id}), "")))
```

Comment:
I think `STAFF_SUBDOMAIN` can be assumed to be present if billing is enabled, at least if we were to add it in `dev_settings.py`, if we think that's cleaner than having this conditional.